app/controllers/partners_controller.rb

```diff
@@ -6,7 +6,8 @@ class PartnersController < ApplicationController
 
   def index
     @unfiltered_partners_for_statuses = Partner.where(organization: current_organization)
-    @partners = Partner.where(organization: current_organization).class_filter(filter_params).alphabetized
+    @partners = Partner.includes(:partner_group).where(organization: current_organization).class_filter(filter_params).alphabetized
+    @partner_groups = PartnerGroup.includes(:partners, :item_categories).where(organization: current_organization)
```

Comment:
that's interesting we need this given the first query for partners. Could we instead do something like `Partner.includes(partner_group: [:item_categories])` and remove `@partner_groups`? Then we could also go through partner to get the same thing, I think

Reply:
Oh I see, we are using `@partner_groups` to give the user a list of groups they can attach a partner to. disregard my comment